app/models/task.py

```diff
@@ -1,5 +1,37 @@
 from app import db
+from datetime import datetime
 
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
```

Comment:
Small note: The default value for nullable when these columns are created is already True, so it does not need to be explicitly stated unless we are making nullable False.